model/layout.md

```diff
@@ -47,21 +50,47 @@ Determines if the layer has fixed or automatic length for its axis which is orth
 
 **Kind**: instance property of [<code>Layout</code>](#Layout)
 
-<a name="Layout+alignment"></a>
-### layout.alignment : <code>String</code>
+<a name="Layout+itemAlignment"></a>
+### layout.itemAlignment : <code>String</code>
 Determines how the children are aligned inside the layer.
 
 When layout direction is `"row"`:
 - `"min"`: Align the children to the top of the layer
 - `"max"`: Align the children to the bottom of the layer 
 - `"stretch"`: Stretch the children to fill the height of the layer 
 - `"center"`: Vertically center the children
+- `"inherit"`: 
+- `"baseline"`: 
 
 When layout direction is `"column"`:
 - `"min"`: Align the children to the left of the layer
 - `"max"`: Align the children to the right of the layer 
 - `"stretch"`: Stretch the children to fill the width of the layer 
 - `"center"`: Horizontally center the children
+- `"inherit"`: 
+- `"baseline"`: 
+
+**Kind**: instance property of [<code>Layout</code>](#Layout)
+
+<a name="Layout+distribution"></a>
+### layout.distribution : <code>String</code>
+Determines how the children are distributed inside the layer.
+- `"center"`: 
+- `"max"`: 
+- `"min"`: 
+- `"space-between"`:
```

Comment:
@SamilCan I need help for these too, thanks!

Reply:
Here the figma's doc about these props: https://www.figma.com/plugin-docs/api/properties/nodes-primaryaxisalignitems/#signature